### ISSUE TITLE
I18n: images with English content

### ISSUE THREAD

**Issue by pocketcasts** [Automattic/pocket-casts-ios#7]
We have images with hardcoded text on it.

### How To Upload View

<img width=300 src="https://user-images.githubusercontent.com/610217/136900247-14bc260e-7051-448f-8a77-5cb06c35413d.PNG" />

Found in: `/podcasts/Files.xcassets`

These images are separated out by theme. A couple of possible options are:

1) Opt to not translate
2) Recreate UI approximations in code
3) Translate a single theme and opt to only use that theme 
4) Translate all themes for all languages

The preferable approach is (2).

**Comment by teaseaque** [comment]
Hey, could you please assign this issue to me? I've forked the repo and worked on a fix in branch [replaced-how-to-upload-images-with-swiftui-views](https://github.com/teaseaque/pocket-casts-ios-how-to-images/tree/replaced-how-to-upload-images-with-swiftui-views)

**Comment by teaseaque** [comment]
@leandroalonso @SergioEstevao @bjtitus can you please help confirm if I can submit a PR to fix this issue as an external contributor? My changes account for included languages and themes, do you recommend any other edits? 

![Image](https://github.com/user-attachments/assets/0e0ee17e-f782-46a7-85dd-9eb989aa972d)

**Comment by bjtitus** [comment]
@teaseaque Absolutely! I'll assign the issue to you. Feel free to submit a PR whenever you have changes. I think we need to create our own copy of your PR to get CI running but we should be able to keep the commits with you as author.

**Comment by teaseaque** [comment]
@bjtitus That is great, thank you! Just submitted PR #2967 and signed the CLA.
